static/app/views/replays/detail/ourlogs/unsupportedReplays.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * SDKs that do not yet support logs in replay
+ */
+const UNSUPPORTED_REPLAY_LOGS_SDK_NAMES = [
+  'sentry.cocoa',
+  'sentry.java.android',
+  'sentry.cocoa.flutter',
+  'sentry.dart.flutter',
+  'sentry.java.android.flutter',
+] as const;
+
+type UnsupportedSDKName = (typeof UNSUPPORTED_REPLAY_LOGS_SDK_NAMES)[number];
+
+export function isLogsUnsupportedBySDK(sdkName: string | null | undefined): boolean {
+  if (!sdkName) {
+    return false;
+  }
```

Comment:
Is this intentional? If sdk name doesn't exist, we return `false` to mean "logs are supported"?